spec/display-buffer-spec.coffee

```diff
@@ -721,6 +733,26 @@ describe "DisplayBuffer", ->
       expect(displayBuffer.screenPositionForBufferPosition([100000, 0])).toEqual [12, 2]
       expect(displayBuffer.screenPositionForBufferPosition([100000, 100000])).toEqual [12, 2]
 
+    it "clips correctly around atomic tab characters", ->
+      #set tabLength higher than 2 to make sure we're not just rounding up; we have to actually clip
```

Comment:
:bulb: Maybe we could be a bit more verbose on this spec, what about:

```coffee
it "clips to the (left or right) edge of an atomic token without rounding up"
```

So that we can get rid of the comment as well :fire: